client/src/includes/tabs.js

```diff
@@ -230,6 +230,13 @@ class Tabs {
         }
       }
     });
+
+    // Dispatch tab-changed on window load
+    window.onload = () => {
+      if (this.tabContainer) {
+        document.dispatchEvent(new CustomEvent('wagtail:tab-changed'));
```

Comment:
Could we dispatch this only one time in `initTabs`, rather than once per set of tabs?

Reply:
Yess good idea!